.gitignore

```diff
@@ -297,3 +297,7 @@ prog
 DISCORD.TOKEN
 sqfvm
 libsqfvm.*
+
+# hidden files
+.*
```

Comment:
this will ignore all files unless one explicitly adds them

Reply:
That is not true. Just checked it and it works as expected.

Refer to [this](https://stackoverflow.com/questions/11543789/git-how-to-ignore-hidden-files-dot-files-files-with-empty-file-names-via-g) or [this](https://stackoverflow.com/questions/8021441/git-how-to-ignore-hidden-directories) as a source for this approach

Reply:
> this will ignore all files unless one explicitly adds them

still valid, please explicitly add rules for the files that shall be ignored (generic as needed)
it is not yet known if there will be `.` prefixed files in the future